src/main/java/com/kainos/ea/controller/WebService.java

```diff
@@ -0,0 +1,41 @@
+package com.kainos.ea.controller;
+
+import com.kainos.ea.dao.RolesDao;
+import com.kainos.ea.database.DatabaseConnection;
+import com.kainos.ea.service.RolesService;
+import io.swagger.annotations.*;
+import org.eclipse.jetty.http.HttpStatus;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Api("Engineering Academy Dropwizard API")
+@Path("/api")
+
+public class WebService {
+
+    private static RolesService rolesService;
+
+    public WebService(){
+        RolesDao rolesDao = new RolesDao();
+        DatabaseConnection databaseConnector = new DatabaseConnection();
+        rolesService = new RolesService(rolesDao, databaseConnector);
+    }
+
+    //End point to get a list of all job roles. us001
```

Comment:
I think we should remove comments that are not needed? I think that goes with other code review I saw, but need second opinion on that

Reply:
Commented code that is not used is probably bad.
Comments that explain functionality are ok in my opinion.

Reply:
this is done